.ci/jenkins/Jenkinsfile

```diff
@@ -65,10 +67,13 @@ def RunDockerContainers(configurationMaps) {
 def PrepareEnvironment(configurationMaps) {
     Closure prepareEnvironmentMethod = { configMap ->
         UTILS.PropagateStatus("Run_docker_containers", configMap["dockerContainerName"])
+        if ( CI_PARAMS.contains("REBUILD NGRAPH") ) {
+            NGRAPH_DIRECTORY="/home"
+            ENVPREP_ARGS="--rebuild-ngraph"
+        }
         sh """
-            docker cp ${CI_ROOT}/prepare_environment.sh ${configMap["dockerContainerName"]}:/home
             docker cp ${CI_ROOT}/utils/docker.sh ${configMap["dockerContainerName"]}:/home
-            docker exec ${configMap["dockerContainerName"]} ./home/prepare_environment.sh
+            docker exec ${configMap["dockerContainerName"]} bash -c "/root/${CI_ROOT}/prepare_environment.sh ${ENVPREP_ARGS}"
```

Comment:
I see that tests are passing but I'm concerned whether this path is correct: `/root/${CI_ROOT}/prepare_environment.sh`  -> which is evaluated (as far as I understand it) into `/root/.ci/jenkins/prepare_environment.sh`. Shouldn't this contain somewhere inside `ngraph` directory ?

Reply:
Yes it should and in fact it does. This is path inside docker container, where root is mapped to directory containing cloned `ngraph-onnx` repo.